main/php_open_temporary_file.c

```diff
@@ -317,9 +326,6 @@ PHPAPI int php_open_temporary_fd_ex(const char *dir, const char *pfx, zend_strin
 	fd = php_do_open_temporary_file(dir, pfx, opened_path_p);
 	if (fd == -1) {
 		/* Use default temporary directory. */
-		if (!(flags & PHP_TMP_FILE_SILENT)) {
-			php_error_docref(NULL, E_NOTICE, "file created in the system's temporary directory");
```

Comment:
Alternatively perhaps just:

> file ***will be*** created […]

Looking at `ext/standard/tests/file/bug52624.phpt` folks might otherwise be confused why the final error message mentions /tmp.

Reply:
I don't think we should change the text completely (even for successful fallback creation) because some people can ignore error message by text in error handler (I saw that in some legacy code in past) so we don't want to break that (or more minimise chance of breaking it) if the target is 8.3. But different message in that TODO bit would make sense.